code/examples/main-qik_2s12v10_c-testing.cpp

```diff
@@ -83,13 +85,49 @@ int main(void) {
                     << '\n';
         hwlib::wait_ms(500);
     }
-
+    if (test_get_motor_current) {
+        hwlib::cout << "Testing the get_m0_current_milliamps function.\n";
+        hwlib::cout << qik_2s12v10_motorcontroller.get_m0_current_milliamps() << '\n';
+        hwlib::cout << "Testing the get_m1_current_milliamps function.\n";
+        hwlib::cout << qik_2s12v10_motorcontroller.get_m1_current_milliamps() << '\n';
+    }
     if (test_get_error) {
         hwlib::cout << "Testing the get_error function.\n";
         hwlib::cout << "Error byte: " << hwlib::bin
                     << qik_2s12v10_motorcontroller.get_error() << hwlib::dec
                     << '\n';
     }
+    if (test_brake_motor){
+        qik_2s12v10_motorcontroller.get_error();         //checking error, because not doing so sometimes causes problems
```

Comment:
Shouldn't this standard? Not on every if statement

Reply:
fixed in https://github.com/R2D2-2019/moving_platform/commit/214aa4ecde851ddaf678ea69d68fe6f29ce738f4 I had forgotten the whole file, it is fixed now.